### DISCUSSION TITLE
How can I get every language supported by prism in my backend?

### DISCUSSION THREAD

**Discussion by nailuj29** [PrismJS/prism#2884] [Q&A]
I'm making a web app that uses a clojure backend, and I would like to be able to get all the languages so I can render a dropdown for them in my backend. Is there like a JSON list I can use?

**Comment by RunDevelopment** [comment]
> Is there like a JSON list I can use?

Yes, [`components.json`](https://github.com/PrismJS/prism/blob/master/components.json) lists all languages, plugins, and themes.

**Reply by nailuj29** [reply]
Does that have the aliases too? It doesn't really matter too much, but would be useful.

**Reply by RunDevelopment** [reply]
Yes, it has everything.
